.sync/workflows/leaf/release-draft.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow automatically drafts new project releases so it is obvious
+# what will a current release will look like at any time.

```

Comment:
first "will" can be dropped.

Reply:
Updated in PR commit https://github.com/microsoft/mu_devops/pull/49/commits/b445c98d20c87a5e36f27d8743aa98b44ca2795d